tests/migration_test.go

```diff
@@ -1168,6 +1171,8 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				if migrationConfiguration != nil {
 					config := getCurrentKv()
 					config.MigrationConfiguration = migrationConfiguration
+					log.Log.Infof("Setting bandwidth migration to %v", config.MigrationConfiguration.BandwidthPerMigration)
```

Comment:
I don't think this log message helps. I would support adding a log message [here](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/live-migration-source.go#L738)

Reply:
Done :+1: